cpp/cmake.md

```diff
@@ -0,0 +1,36 @@
+# CMake
+CMake is the de facto standard build system for C++ projects.
+
+Here are a collection of recommended links, guides and talks on how to use it.
+
+## Getting Started
+We recommend this guide to get started with CMake:
+
+- [An Introduction to Modern CMake](https://cliutils.gitlab.io/modern-cmake/)
+
+Or alternatively these two videos from Jason Turner:
+
+- [Intro to CMake](https://www.youtube.com/watch?v=HPMvU64RUTY)
+- [The Ultimate CMake / C++ Quick Start](https://www.youtube.com/watch?v=YbgH7yat-Jo&t=0s)
+
+It can also be helpful to take a look at a simple example CMake project:
+
+- [CLIUtils/modern-cmake/simple-project](https://gitlab.com/CLIUtils/modern-cmake/-/tree/master/examples/simple-project)
+- [pabloariasal/modern-cmake-sample](https://github.com/pabloariasal/modern-cmake-sample)
+- [ssciwr/cpp-project-template](https://github.com/ssciwr/cpp-project-template)
+
+## Official Documentation
+The official CMake [documentation](https://cmake.org/cmake/help/latest/) is a very useful and complete source of reference information,
+for example if you want to look up the specific details of a particular command. It can also be a bit overwhelming, and
+doesn't have many examples of how to actually do things, so it is probably not the best place to start learning how to use CMake.
+
```

Comment:
It could be worth to add a comment that the written book "Mastering CMake" is not a good starting point either.

Reply:
I think in general we should try to stick to recommending helpful resources rather than saying which ones we don't think are so good - but since "read the manual" is often good advice to start with I thought I should put this warning here.